mne/viz/tests/test_topomap.py

```diff
@@ -329,8 +329,6 @@ def get_texts(p):
     # make sure projector gets toggled
     assert (np.max(fig1.axes[0].images[0]._A) != data_max)
 
-    pytest.raises(RuntimeError, plot_evoked_topomap, evoked,
-                  np.repeat(.1, 50), time_unit='s')
```

Comment:
Rather than eliminate this entirely, it's a warning now, right? So you could do:
```
with pytest.warns(RuntimeWarning, match='some regex substring to match'):
    plot_evoked_topomap(evoked, [0.1] * 26, res=4)
```
But if it adds a bunch of time to the test (please do check) then it's not worth it just to catch the one warning line

Reply:
Just added that.

Reply:
It added 0.5 sec to the test which is pretty rough for something so simple to test (our tests are already quite slow). Every little bit helps, so I pushed a commit using `monkeypatch` to speed things up.